apps/angular/src/app/features/search-form/search-form.component.html

```diff
@@ -0,0 +1,13 @@
+<form
+	class="search-form"
+	[formGroup]="form"
+	(ngSubmit)="onSubmit()"
+>
+	<mat-form-field appearance="outline" class="search-field">
+		<mat-label class="search-title">Search by title</mat-label>
+		<input matInput placeholder="One Piece" type="search" class="search-input" formControlName="search" [value]="value">
```

Comment:
You don't need to bind the [value] of  `input`, the form control already supported that as you set the initial value in `FormControl`
```suggestion
		<input matInput placeholder="One Piece" type="search" class="search-input" [formControl]="searchControl">
```